src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexPersistence.cs

```diff
@@ -172,6 +235,39 @@ internal LuceneIndexWriter EnsureIndexWriter(IState state)
             }
         }
 
+        internal Dictionary<string, LuceneSuggestionIndexWriter> EnsureSuggestionIndexWriter(IState state)
+        {
+            if (_suggestionsIndexWriters != null)
+                return _suggestionsIndexWriters;
+            
+            _suggestionsIndexWriters = new Dictionary<string, LuceneSuggestionIndexWriter>();
+
+            foreach (var item in _fields)
+            {
+                if (!item.Value.HasSuggestions)
+                    continue;
+
+                string field = item.Key;
+                
+                try
+                {
+                    var snapshotter = new SnapshotDeletionPolicy(new KeepOnlyLastCommitDeletionPolicy());
+                    // TODO [ppekrol] support for IndexReaderWarmer?
+                    var writer = new LuceneSuggestionIndexWriter(field, _suggestionsDirectories[field], 
+                                        snapshotter, IndexWriter.MaxFieldLength.UNLIMITED, 
+                                        _index._indexStorage.DocumentDatabase, state);
+
+                    _suggestionsIndexWriters[field] = writer;
+                }
+                catch (Exception e)
+                {
+                    throw new IndexWriteException(e);
```

Comment:
You are throwing specific exception, but aren't adding any additional information on top of it.